src/main/java/com/meilisearch/sdk/Documents.java

```diff
@@ -158,7 +197,12 @@ TaskInfo deleteDocument(String uid, String identifier) throws MeilisearchExcepti
      * @throws MeilisearchException if the client request causes an error
      */
     TaskInfo deleteDocuments(String uid, List<String> identifiers) throws MeilisearchException {
-        String urlPath = "/indexes/" + uid + "/documents/" + "delete-batch";
+        URLBuilder urlb = new URLBuilder();
+        urlb.addSubroute("indexes")
+                .addSubroute(uid)
+                .addSubroute("documents")
+                .addSubroute("delete-batch");
+        String urlPath = urlb.getURL();
```

Comment:
There already is a `toQuery` method as `public String toQuery(String uid)`
I can modify it to return the `urlb` instead of the URL directly to be able to add this:
`.addSubroute("delete-batch");`
But I don't think it will be better